base/applications/rapps/settingsdlg.cpp

```diff
@@ -25,24 +39,24 @@ ChooseFolder(HWND hwnd)
     bi.ulFlags =
         BIF_USENEWUI | BIF_DONTGOBELOWDOMAIN | BIF_RETURNONLYFSDIRS | /* BIF_BROWSEFILEJUNCTIONS | */ BIF_VALIDATE;
 
-    if (SUCCEEDED(CoInitializeEx(NULL, COINIT_APARTMENTTHREADED)))
+    if (SUCCEEDED(CoInitialize(NULL)))
```

Comment:
It's probably better to keep the original Ex code with the appartment thread flag explicit.

Reply:
CoInitialize is always STA and shell functions are STA.